go/ql/test/library-tests/semmle/go/Function/TypeParamType.expected

```diff
@@ -13,3 +13,4 @@
 | TG2 | interface { } |
 | U | interface { } |
 | V | interface { int64 \| float64 } |
+| bytes | interface { []uint8 \| string } |
```

Comment:
I guess this is coming from a new library function?

Reply:
It appears that way, yes